src/sdk/models/policies.ts

```diff
@@ -17,3 +18,27 @@ export interface PolicyJob extends Job {
 export interface PolicyJobOptions extends TransferPath {
 	policies?: Policy[];
 }
+
+export interface PolicyItem extends IEntityIdentifier<number> {
+	parent_id?: number;
+	source?: TransferPlatformItem;
```

Comment:
Sorry, I missed this but the tracking model for policy items will have these inlined rather than under the `source` prop.